wp-content/themes/borderzine/less/typography.less

```diff
@@ -9,6 +10,15 @@ a {
   }
 }
 
+button {
+  font-family: @body;
+}
+
+.btn-primary,
+.btn-primary:hover {
+  background-color: @orange;
+}
```

Comment:
We need to have some visual difference here for the :hover and :focus states from the inactive state. Can we use a different background color for that, or a white border on the element?

Reply:
@benlk https://github.com/INN/umbrella-borderzine/pull/62/commits/09fae219522e02863bf219ca7690b674fcd0f282 uses `#2e4052` as the `btn-primary:focus` background color. `#2e4052` is the color of their donate button (that turns orange on hover) on their prod site.

Reply:
Why not also use that color for `:hover`?

Reply:
@benlk a few days ago @kaylima mentioned the hover state background color for buttons was fine as-is, so I didn't want to change it.